### DISCUSSION TITLE
(Ideas) Split application proxying for windows 10/11

### DISCUSSION THREAD

**Discussion by glory7sa** [hiddify/hiddify-app#1406] [Ideas]
Separate proxy connection tunneling of individual applications via VPN for windows 10/11
https://assets-cms.privateinternetaccess.com/photos/shares/pia-steps/Split-step3.png

**Comment by glory7sa** [comment]
https://ibb.co/DC7pNdW

**Reply by Korb** [reply]
> https://ibb.co/DC7pNdW

Original:
![image](https://github.com/user-attachments/assets/6ae0c25c-5b68-43c0-a096-b90ba15914a1)

Google Translate:
![2025-02-15_19-11-52](https://github.com/user-attachments/assets/e6ba0af5-fa53-4eba-9213-b048c8fd4bfd)

Am I correct in understanding that the feature called "**Split tunneling** (Allows you to connect to some sites or applications through a VPN connection and bypass others)" in Amnezia VPN is currently missing from Hiddify for Windows? This feature may exist in two sub-features - "**Site-based** split tunneling" and "**App-based** split tunneling", and both options are useful if you are using the app in a country with heavy censorship.

Upd. ~  "[Per-site proxy](https://github.com/hiddify/hiddify-app/discussions/1265)" Q&A Discussion.

**Comment by lynrayy** [comment]
bump
